cli/pkg/config/happy_config.go

```diff
@@ -275,8 +281,18 @@ func (s *HappyConfig) GetFeatures() *Features {
 	return &s.getData().FeatureFlags
 }
 
-func (s *HappyConfig) GetHappyApiBaseUrl() string {
-	return s.getData().HappyApiBaseUrl
+func (s *HappyConfig) GetHappyApiConfig() HappyApiConfig {
+	apiConfig := s.getData().Api
+	if apiConfig.BaseUrl == "" {
+		apiConfig.BaseUrl = "https://hapi.hapi.prod.si.czi.technology"
+	}
+	if apiConfig.OidcClientID == "" {
+		apiConfig.OidcClientID = "0oa7owjlihple45jJ5d7"
```

Comment:
instead of hardcoding, I wonder if we should just fail if these values aren't set to begin with and auth is required